src/components/PeopleFilters.tsx

```diff
@@ -22,74 +54,56 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={`${query || ''}`}
```

Comment:
```suggestion
            value={query}
```

Reply:
ERROR in src/components/PeopleFilters.tsx:59:13
TS2322: Type 'string | null' is not assignable to type 'string | num'.
    57 |             className="input"
    58 |             placeholder="Search"
  > 59 |             value={query}
       |             ^^^^^
    60 |             onChange={handleQueryChange}
    61 |           />
    62 |